iroh/src/rpc_protocol/authors.rs

```diff
@@ -0,0 +1,146 @@
+use iroh_base::rpc::RpcResult;
+use iroh_docs::{Author, AuthorId};
+use quic_rpc::message::{Msg, RpcMsg, ServerStreaming, ServerStreamingMsg};
+use serde::{Deserialize, Serialize};
+
+use super::RpcService;
+
+#[allow(missing_docs)]
+#[derive(strum::Display, Debug, Serialize, Deserialize)]
+#[nested_enum_utils::enum_conversions(super::Request)]
+pub enum Request {
+    List(AuthorListRequest),
```

Comment:
given this is scoped now, I would remove the `Author` prefix

Reply:
Did it for all messages. There are occasionally some ambiguities now, requiring import as..., but they will go away once the handlers are also separated...